### PR TITLE
Limit Tracking Update

### PR DESCRIPTION
**Pull request by sojanpr**
Adding support to limited tracking

@dmitrig01 @derrickstaten

### CODE REVIEW

**Comment by E-B-Smith** [comment]
The iOS version is https://github.com/BranchMetrics/ios-branch-deep-linking/pull/754

**Comment by dmitrig01** [comment]
I can't give a great review, but it looks fine to me

File: Branch-SDK/src/io/branch/referral/Defines.java [hunk 1]
```diff
@@ -67,6 +67,7 @@
         AAID("aaid"),
         LATVal("lat_val"),
         LimitedAdTracking("limit_ad_tracking"),
+        limitFacebookTracking ("limit_facebook_tracking "),
```
Comment (dmitrig01) [inline Branch-SDK/src/io/branch/referral/Defines.java]: extra space

**Comment by sojanpr** [comment]
Test steps 
1) Add `Branch.getInstance().setLimitFacebookTracking(true)` in `MainActivity#onCreate()` method.
2) Open application and create few v1 and v2 api calls
Expected result
For v1 requests there should be `limit_facebook_tracking` added to the request body
For v2 requests there should be `limit_facebook_tracking` added to the user_data.
There should not be  `limit_facebook_tracking` added to the request  if `setLimitFacebookTracking()` not called

**Pull request merged**